recipients-resolver/src/main/java/com/redhat/cloud/notifications/recipients/resolver/RecipientsResolver.java

```diff
@@ -134,16 +147,15 @@ private static Optional<Set<String>> extractRequestUsersIntersection(Set<Recipie
         }
     }
 
-    private Optional<Set<String>> callKessel() {
-        Optional<Set<String>> allowedUsers = Optional.empty();
-        try {
-            ApiRebacV1ReadRelationshipsResponse kesselResponse = kesselRelationshipApi.relationshipsReadRelationships("group", "bob_club", "member", null, null, null);
-            Log.infof("Kessel Open Api response: %s", kesselResponse);
-            allowedUsers = Optional.of(kesselResponse.getRelationships().stream().filter(rel -> "user".equals(rel.getSubject().getObject().getType())).map(rel -> rel.getSubject().getObject().getId().toLowerCase()).collect(Collectors.toSet()));
-            Log.infof("Allowed users from Kessel are: %s", allowedUsers);
-        } catch (Exception ex) {
-            Log.error("Error calling Kessel relationship Api");
+    private Set<String> callKessel(ExternalAuthorizationCriteria externalAuthorizationCriteria) {
```

Comment:
What do you think about naming this function as `findUsersWithCriteria`?

Reply:
Ok